### ISSUE TITLE
[Please help] Can't get it to work on windows 10 with python 3

### ISSUE THREAD

**Issue by Pawloland** [dtmilano/AndroidViewClient#274]
The release v20.0.0.b3 is supposed to be working wit python 3.6+. I have python 3.8.0 installed.
What I have set up:

- python 3.8.0 is installed on windows 10
- python is in the system 'path' variable (can enter python shell from any directory by typing python in PowerShell)
- platform-tools folder containing `adb.exe` and `fastboot.exe` is in the system `path` variable

At first I thought the setup would be as simple as typing `pip install androidviewclient` in cmd. I realized that using this command, version 15.8.1 (incompatible with python 3) is downloaded. Then I run `pip uninstall androidviewclient` and `pip install androidviewclient==20.0.0b3` to install version which supports python 3.

Here is the problem:
The guide on github is made based on Linux system, not windows. What I have understood, AndroidViewClient is made out of couple of parts, mainly `culebra` and `dump`.
If I understood correctly, they work just like any other command in Linux for example, `ls`, `cp` or `mv`, they have their own arguments and options. 

- When I run  `dump --help` or `culebra --help` on windows, in PowerShell I get error, which says that there is no internal or external command such as `dump`, or in the next case scenario `culebra`.
- When I run `python culebra --help` or `python dump --help` I get error:
`can't open file 'dump': [Errno 2] No such file or directory`.

Is it supposed to be like that on windows? 

What I am trying to achieve with AndroidViewClient is to add an option to my program, which will open specified app and then click button, on every device connected, just like in this video: https://www.youtube.com/watch?v=yKnpdlospL4 (but if it is possible, make it run much faster), or only on selected ADB device or devices.

Thank in advanced.

**Comment by dtmilano** [comment]
This may help you: https://darpan.blog/code/setup-androidviewclient-windows/

It is for python 2.7 but the main concept would apply.

**Comment by Pawloland** [comment]
I managed to make culebra and dump work on windows 10 with python 3.8.0. What I did was:

- downloaded zip from github,
- extracted zip to "AndroidViewClient-master" folder,
- located "AndroidViewClient-master\tools" folder containing `culebra` and `dump` files,
- realized, that those files are in fact just python scripts without `.py` extinction,
- run `python culebra --help` and `python dump --help` while being in the directory "AndroidViewClient-master\tools" without the error.

I am not sure though if it is desired setup path to be followed and thus, there are my new questions:

- Are `culebra` and `dump` tools supposed to be working on windows, while executing command `python culebra` or `python dump` from any directory, just after installing AndroidViewClient v20.0.0.b3 using pip?
- If they aren't supposed to be working out of the box, after pip installation method, can those two files: `culebra` and `dump` be moved from "AndroidViewClient-master\tools" to any different directory? I am asking because I don't know if their functionality is based on different files in "AndroidViewClient-master" directory. If they doesn't depend, I would like to make `culebra ` and `dump`  work like normal windows command: from any directory and without the need to type `python culebra` all the times but just `culebra`.
